packages/website/next.config.js

```diff
@@ -4,5 +4,5 @@ const withMDX = require('@next/mdx')({
 });
 
 module.exports = withMDX({
-  pageExtensions: ['tsx', 'mdx'],
+  pageExtensions: ['ts', 'tsx', 'mdx'],
```

Comment:
This PR doesn't add any `.ts` pages, are you sure this is necessary?

Reply:
yes, files under api folder are `.ts`

Reply:
Yeah but they don't render as pages anyway.

Reply:
NextJs ignores files which extensions are not in this array